apps/teams-test-app/src/components/AppAPIs.tsx

```diff
@@ -16,37 +16,20 @@ const GetContext = (): ReactElement =>
 const ExecuteDeepLink = (): ReactElement =>
   ApiWithTextInput<string>({
     name: 'executeDeepLink2',
```

Comment:
Do these instances of ExecuteDeepLink and executeDeepLink2 need to be renamed as well?

Reply:
I renamed the component, but renaming executeDeepLink2 will break the e2e tests until I make the same change in the hub sdk. I think it's fine as is, since it's just an ID used for those tests, but I'll rename it if we think we should

Reply:
Along the same lines, I didn't change the openLink SDK message from 'executeDeepLink'. Do we want those to match the API name?

Reply:
The test one makes sense, @MarcinJuraszek  mentioned something like this when I asked a few weeks ago. Might be able to check with him if he thinks it's worth updating the tests.

Not sure about the message name though

Reply:
The openLink SDK message needs to rename 'executeDeepLink' for back compat over-the-wire message passing.